api/test/unit/controllers/test_search_controller_search_query.py

```diff
@@ -0,0 +1,240 @@
+    settings.USE_RANK_FEATURES = False
+    serializer = media_type_config.search_request_serializer(data={})
+    serializer.is_valid()
+    search_query = search_controller.create_search_query(serializer)
+    actual_query_clauses = search_query.to_dict()["bool"]
+
+    assert actual_query_clauses == {
+        "must_not": [{"term": {"mature": True}}],
+        "must": [{"match_all": {}}],
+    }
+
+
+def test_create_search_query_q_search_no_filters(media_type_config):
+    serializer = media_type_config.search_request_serializer(data={"q": "cat"})
+    serializer.is_valid()
+    search_query = search_controller.create_search_query(serializer)
+    actual_query_clauses = search_query.to_dict()["bool"]
+
+    assert actual_query_clauses == {
+        "must_not": [{"term": {"mature": True}}],
+        "must": [
+            {
+                "simple_query_string": {
+                    "default_operator": "AND",
+                    "fields": ["title", "description", "tags.name"],
+                    "query": "cat",
+                }
+            }
+        ],
+        "should": [
+            {
+                "simple_query_string": {
+                    "boost": 10000,
+                    "fields": ["title"],
+                    "query": "cat",
+                }
+            },
+            {"rank_feature": {"boost": 10000, "field": "standardized_popularity"}},
+        ],
+    }
+
+
+def test_create_search_query_q_search_with_quotes_adds_exact_suffix(media_type_config):
+    serializer = media_type_config.search_request_serializer(
+        data={"q": '"The cutest cat"'}
+    )
+    serializer.is_valid()
+    search_query = search_controller.create_search_query(serializer)
+    actual_query_clauses = search_query.to_dict()["bool"]
+
+    assert actual_query_clauses == {
+        "must_not": [{"term": {"mature": True}}],
+        "must": [
+            {
+                "simple_query_string": {
+                    "default_operator": "AND",
+                    "fields": ["title", "description", "tags.name"],
+                    "query": '"The cutest cat"',
+                    "quote_field_suffix": ".exact",
+                }
+            }
+        ],
+        "should": [
+            {
+                "simple_query_string": {
+                    "boost": 10000,
+                    "fields": ["title"],
+                    "query": "The cutest cat",
+                }
+            },
+            {"rank_feature": {"boost": 10000, "field": "standardized_popularity"}},
+        ],
+    }
+
+
+def test_create_search_query_q_search_with_filters(image_media_type_config):
+    serializer = image_media_type_config.search_request_serializer(
+        data={
+            "q": "cat",
+            "license": "by-nc",
+            "aspect_ratio": "wide",
+            # this is a deprecated param, and it doesn't work because it doesn't exist in the serializer
+            "categories": "digitized_artwork",
```

Comment:
Are we including it here just to showcase that?

Reply:
I included it in the PR to showcase this to the reviewers, but I should probably remove it from the test that will be merged.